packages/gatsby/src/query/error-parser.ts

```diff
@@ -143,6 +143,7 @@ const errorParser = ({
     if (matched) {
       structured = {
         ...cb(matched),
+        category: `USER`,
```

Comment:
As mentioned on Slack, not every error should get that category but be added individually in the error map (in the same file above this line)

Reply:
What should these errors be categorized as? I assumed they would be `USER` errors since they largely seem to be errors caused by bad GraphQL syntax.

Should we create a `GRAPHQL_SYNTAX` category?

I pushed up a change so that only the target error I was after is categorized as `USER`, but the rest will still be uncategorized errors :/

Reply:
Nevermind! 

@KyleAMathews mentioned this error is actually happening more as a result of a bug. For now, we can just categorize the target error ("Unable to create") as a `USER` error.

This will be in play until the underlying bug is fixed. :+1: